packages/angular_devkit/build_angular/src/utils/delete-output-dir.ts

```diff
@@ -18,5 +18,11 @@ export function deleteOutputDir(root: string, outputPath: string): void {
     throw new Error('Output path MUST not be project root directory!');
   }
 
-  rmdirSync(resolvedOutputPath, { recursive: true, maxRetries: 3 });
+  try {
+    rmdirSync(resolvedOutputPath, { recursive: true, maxRetries: 3 });
```

Comment:
The `recursive` option is deprecated in Node.js v16 (which I think will cause a warning to be shown on the console).  `rmdirSync` in v16 will also throw on a nonexistent directory.
The `fs.rmSync` function is the recommended replacement which was introduced in v14.14.  I think we should probably adjust this to detect it and try `rmSync(resolvedOutputPath, { force: true, recursive: true, maxRetries: 3 });` first and then fallback to the existing function if the new one is not found.  Once v12 support is dropped, this could be cleaned up to just the new function.

Reply:
Let’s do that.